src/models/Collection.js

```diff
@@ -103,6 +103,22 @@ class Collection {
     };
   }
 
+  static async getParents(noid) {
+    let rows;
+    try {
+      rows = await viewResultsFromKeys("collection", "access", "items", [noid]);
+    } catch (error) {
+      throw error;
+    }
+    let items = {};
+    rows.map((row) => {
+      items[row.id] = row.value;
+      row.value.slug = row.value.slug;
+      row.value.label = multiTextValueToSingle(row.value.label);
+    });
+    console.log("check parents", items);
+    return items;
+  }
```

Comment:
If i have to return the items similar to the loadParents method, is it ok to change the existing function to static and pass a noid to it?
The way i have iterated will result in  something like:
id ="id"
slug ="slug"
label="label"
i could do a return{
id:id
slug:slug
label:label
}
Which is a right way is one question. 
Next is where should the getPArents method be called?